website-builds.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+REF="$1"
+TAG="website-builds:$REF"
+
+if [ -z "$REF" ]; then
+    echo >&2 "Usage: $0 ref"
+    exit 1
+fi
+
+if ! which docker >/dev/null 2>&1; then
+   echo >&2 "Docker is required to run this script."
+   echo >&2 "https://docs.docker.com/get-docker/"
+   exit 1
+fi
+set -e
+
+docker build --no-cache -t "$TAG" -<<EOF
+FROM nginx:alpine
+
+RUN cd /usr/share/nginx \
+ && wget "https://github.com/certbot/website-builds/archive/$REF.zip" \
+ && unzip *.zip \
+ && mv website-builds-*/* html
+EOF
+
+echo
+echo "Starting server. You can access the website in your browser at http://localhost:8000. Press Ctrl-C to shut down."
```

Comment:
this gets kind of lost since the docker/server output pretty much immediately kicks it off the screen, but I don't actually know if there's a good solution to that -- we probably actually do want that output visible.

Reply:
I thought it'd be nice to put some output here anyway since the container doesn't emit any when it first comes up, to signal that the build has finished and the server is running. I think only the shutdown instruction is really lost, since if nginx logs kick the message off the screen that means the user is accessing http://localhost:8000 at least. I moved the note about Ctrl-C to a new line and added some escape codes to color the output... Does that work? Is it too much?

Reply:
I actually love the colored output, very official-looking. On mine the container is emitting stuff, that's the problem -- the last thing I get is `/docker-entrypoint.sh: Configuration complete; ready for start up`. I wonder if we can just pass that print statement to the docker container, so it prints it once it's done with startup? If not it's not the biggest deal, this is definitely much better, but the `localhost:8000` info is getting lost as well.
![Screenshot from 2021-01-13 14-36-19](https://user-images.githubusercontent.com/1227205/104518568-b8e9d480-55ac-11eb-8ae5-8b10da328161.png)

Reply:
Ooh ok. Sorry, my nginx:alpine image was 21 months old! I pulled the latest and now I see the additional output. The image let's you silence the entrypoint messages with an environment variable, which I've set. How's it look to you now?

Reply:
Ah, that explains it! Much better, thanks!!